src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -83,6 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $data['tests'] = array_keys($data['tests']);
             $data['loader_paths'] = $this->getLoaderPaths();
             $io->writeln(json_encode($data));
+            $this->displayAlternatives($this->findWrongBundleOverrides(), $io);
```

Comment:
It'll break the json output, maybe should be included into a new `$data['warning']` above?

Reply:
Done. Thank you